src/backend/aspen/util/pathogen_configs.py

```diff
@@ -0,0 +1,15 @@
+from typing import Any, Dict
+
+
+def get_lineage_urls(pathogen_slug: str) -> Dict[str, Any]:
```

Comment:
Since the plan is for this to eventually build up to supporting many pathogen-lineages down the road, I'd like to see a some documentation on what the intent here with the structure is. There's some assumption about how the lineages data will be structured, and while I can figure it out by cross-referencing `format_lineage_data` where the path/keys stuff gets used, I'd like to see it written out explicitly. As much to make it easier to crank them out when we add more pathogen-lineages as to help see where to extend the structure when we encounter a pathogen-lineages info structure that's totally foreign to this one.

That said, I think having that documentation over in `format_lineage_data` makes an equal amount of sense. I just want it somewhere and a comment on the other one saying "yo, go look at the docs there to understand the usage structure".